tools/respec2html.js

```diff
@@ -13,156 +15,90 @@ colors.setTheme({
-    name: "debug",
-    type: Boolean,
-  },
-  {
-    default: false,
-    description: "Log processing status to stdout.",
-    name: "verbose",
-    type: Boolean,
-  },
-];
+const cli = sade("respec [source] [destination]", true)
+  .describe("Converts a ReSpec source file to HTML and writes to destination.")
+  .example(`input.html output.html ${colors.dim("# Output to a file.")}`)
+  .example(
+    `http://example.com/spec.html stdout ${colors.dim("# Output to stdout.")}`
+  )
+  .example(
+    `http://example.com/spec.html output.html -e -w ${colors.dim(
+      "# Halt on errors or warning."
+    )}`
+  )
+  .example("--src http://example.com/spec.html --out spec.html");
 
-const usageSections = [
-  {
-    header: "respec2html",
-    content: "Converts a ReSpec source file to HTML and prints to std out.",
-  },
-  {
-    header: "Options",
-    optionList,
-  },
-  {
-    header: "Examples",
-    content: [
-      {
-        desc: "1. Output to a file. ",
-        example:
-          "$ ./respec2html.js --src http://example.com/spec.html --out spec.html",
-      },
-      {
-        desc: "2. Halt on errors or warning ",
-        example:
-          "$ ./respec2html.js -e -w --src http://example.com/spec.html --out spec.html",
-      },
-    ],
-  },
-  {
-    content: "Project home: {underline https://github.com/w3c/respec}",
-    raw: true,
-  },
-];
+cli
+  // For backward compatibility
+  .option("-s, --src", "URL to ReSpec source file.")
+  // For backward compatibility
+  .option("-o, --out", "Path to output file.")
+  .option(
+    "-t, --timeout",
+    "How long to wait before timing out (in seconds).",
+    10
+  )
+  .option("-e, --haltonerror", "Abort if the spec has any errors.", false)
+  .option("-w, --haltonwarn", "Abort if ReSpec generates warnings.", false)
+  .option("--disable-sandbox", "Disable Chromium sandboxing if needed.", false)
+  .option("--devtools", "Enable debugging and show Chrome's DevTools.", false)
+  .option("--verbose", "Log processing status to stdout.", false);
 
-(async function run() {
-  let parsedArgs;
-  try {
-    parsedArgs = commandLineArgs(optionList);
-  } catch (err) {
-    console.info(getUsage(usageSections));
-    console.error(colors.error(err.message));
-    return process.exit(127);
-  }
-  if (!parsedArgs.src) {
-    console.info(getUsage(usageSections));
-    return process.exit(2);
+cli.action((source, destination, opts) => {
+  if (!source) {
+    if (!opts.src) {
+      cli.help();
+      process.exit(1);
+    }
+    const msg =
+      `-s / --src option is deprecated in favor of [source] positional argument.` +
+      ` See --help for new syntax.`;
+    console.warn("⚠", colors.yellow(msg));
+    source = opts.src;
   }
-  if (parsedArgs.help) {
-    console.info(getUsage(usageSections));
-    return process.exit(0);
+  if (!destination && opts.out) {
+    const msg =
+      `-o / --out option is deprecated in favor of [destination] positional argument.` +
```

Comment:
as above... supporting the explicit means seems fine. Not something we would remove in the long term.